local-cli/runIOS/runTVOS.js

```diff
@@ -0,0 +1,20 @@
+/**
+* Copyright (c) 2015-present, Facebook, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree. An additional grant
+* of patent rights can be found in the PATENTS file in the same directory.
+*/
+'use strict';
+
+const RunIOSInternals = require('./RunIOSInternals');
+
+const run_ios = new RunIOSInternals('appletvos',
+                                    'appletvsimulator',
+                                    '-tvOS',
+                                    'tvOS',
+                                    'run-tvos',
+                                    'Apple TV 1080p');
+
+module.exports = run_ios.makeExports();
```

Comment:
Can this be `module.exports = makeRunCommand(opts)` instead of a class and `makeExports()`? 

I believe a simple closure that returns an object would be enough:
```js
function makeRunCommand(opts: Options) {
   return (argv, config, args) => {
      // stuff here
   };
}